classes/class-wc-connect-service-settings-store.php

```diff
@@ -110,6 +110,45 @@ public function can_user_manage_payment_methods() {
 				( is_a( $master_user, 'WP_User' ) && $current_user->ID === $master_user->ID );
 		}
 
+		public function get_end_shipper_email() {
+			$master_user = WC_Connect_Jetpack::get_master_user();
+
+			if ( $master_user instanceof WP_User ) {
+
+				$connected_data = WC_Connect_Jetpack::get_connected_user_data( $master_user->ID );
+
+				if ( is_array( $connected_data ) && ! empty( $connected_data['email'] ) ) {
+					return $connected_data['email'];
+				}
+			}
+
+			return '';
+		}
+
+		/**
+		 * Gets the store's location settings.
+		 *
+		 * @return array
+		 */
+		public function get_end_shipper_address() {
+			$store_settings = array(
+				'company'   => html_entity_decode( get_bloginfo( 'name' ), ENT_QUOTES ), // HTML entities may be saved in the option.
```

Comment:
Should we use the site name as the company name? Maybe we should also use the company name from the origin form? cc @laurajohnsonwoo @harriswong

Reply:
As the Company Name is optional, if it's in the Origin Form then I would like to use that, but if not, the merchant's name should suffice. Do we always have that though? 

I hesitate to use the site name as it might not be accurate, but it's a good backup option if we need to use it!

Reply:
Maybe we need a new section inside settings to allow endshipper-specific info as a follow-up ticket? By default, we will fall back to "what Woo thinks" is the best. But with the settings page, it will provide an option to overwrite our guess.